sootup.java.core/src/test/java/sootup/java/core/printer/LegacyJimplePrinterTest.java

```diff
@@ -157,4 +161,17 @@ public void testValidOptions() {
         new JimplePrinter(JimplePrinter.Option.UseImports, JimplePrinter.Option.LegacyMode);
     p.printTo(buildClass(Body.builder()), new PrintWriter(new StringWriter()));
   }
+
+  @Test
+  public void testLegacyEscaping() {
+    StringWriter out = new StringWriter();
+    PrintWriter writer = new PrintWriter(out);
+    JimplePrinter printer = new JimplePrinter(JimplePrinter.Option.LegacyMode);
+
+    SootClass clazz = buildClass(Body.builder(), "dummyMain", "from");
+    printer.printTo(clazz, writer);
+    String jimple = out.toString();
+    assertEquals("public class dummyMain\n{\n    public static void \'from\'()\n    {\n    }\n}\n\r\n", jimple);
```

Comment:
shouldn't it expect it without escaped quotes?
```suggestion
    assertEquals("public class dummyMain\n{\n    public static void 'from'()\n    {\n    }\n}\n\r\n", jimple);
```

Reply:
Yes, you are right.